packages/core/package.json

```diff
@@ -15,11 +15,13 @@
     "/loader"
   ],
   "scripts": {
-    "build": "stencil build --docs-readme",
-    "start": "stencil build --watch --serve --no-open",
-    "stencil": "stencil",
-    "generate": "stencil generate",
-    "storybook": "start-storybook --static-dir ./dist --port 56106"
+    "clear": "rimraf dist && rimraf loader",
```

Comment:
waarom moeten deze zaken steeds weg?

Reply:
Anders wacht de `wait-on` uit `scripts/start.js` en `scripts/cypress.js` niet: Die files zijn er immers dan al, met `wait-on` wacht ik tot die files er zijn en dan pas gaat ie storybook starten.

En daarnaast, het is altijd wel een goed idee om met een schone buildoutput te beginnen